MUXControls.sln

```diff
@@ -1050,20 +1058,25 @@ Global
 		dev\SwipeControl\SwipeControl.vcxitems*{fd63f767-bd98-45d9-8a63-3cd4a148d527}*SharedItemsImports = 9
 	EndGlobalSection
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug_test|Any CPU = Debug_test|Any CPU
```

Comment:
Remove the "Any CPU" sln configuration, since we don't build for that target. You can do this through Build -> Configuration Manager in VS.

Reply:
Addressed by https://github.com/microsoft/microsoft-ui-xaml/pull/3212/commits/92e637aa12533fea4f5c244004987c6d3a7608e0